OWNERS_ALIASES

```diff
@@ -25,13 +25,11 @@ aliases:
   - tao12345666333
 
   ingress-nginx-helm-maintainers:
-  - ubergesundheit
 
   ingress-nginx-helm-reviewers:
-  - ubergesundheit
```

Comment:
We should remove the whole group as I'm not sure if an empty group works. Also I think a key without a value is interpreted as an object / map in YAML.

Reply:
yea i wasnt sure about it either. I can delete the whole thing.